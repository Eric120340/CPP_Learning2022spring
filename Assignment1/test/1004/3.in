4 4 5
11 44 49 21 
18 46 25 16 
30 32 31 5 
31 50 43 49 
