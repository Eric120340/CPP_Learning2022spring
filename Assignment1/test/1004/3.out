445546628 430090158 831990707 814271340 
805123259 211267521 843220804 265486386 
338052770 325924312 124163833 857243464 
854940312 987067342 526275099 888278838 
